CMakeSettings.json

```diff
@@ -29,6 +29,52 @@
           "value": "${workspaceRoot}\\Externals\\Qt\\Qt5.14.1\\msvc2019_64\\lib\\cmake\\Qt5"
         }
       ]
+    },
+    {
+      "name": "Release (arm64)",
+      "configurationType": "Release",
+      "generator": "Ninja",
+      "inheritEnvironments": [ "msvc_arm64_x64" ],
+      "buildCommandArgs": "",
+      "buildRoot": "${workspaceRoot}\\Build\\${name}",
+      "cmakeCommandArgs": "",
+      "variables": [
+        {
+          "name": "Qt5_DIR",
+          "value": "${workspaceRoot}\\Externals\\Qt\\Qt5.14.1\\msvc2019_arm64\\lib\\cmake\\Qt5"
+        },
+        {
+          "name": "CMAKE_SYSTEM_NAME",
```

Comment:
Does this need to be explicitly set, or can it come from the toolchain?

Reply:
Yes it has to be, otherwise the other argument gets ignored as well.